scripts/dataset_utils

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import click
+
+from chimedb.core import connect as connect_db, close as close_db
+from chimedb.dataset.get import Dataset, DatasetCache, index
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+@click.argument("dataset_id")
+def treesize(dataset_id):
```

Comment:
This should have a docstring such that it gives a help message saying what it does.